### ISSUE TITLE
Refactor templates 

### ISSUE THREAD

**Issue by matthewbentley** [beta-nu-theta-chi/ox-dashboard#14]
TODO: write a todo list for this

**Comment by matthewbentley** [comment]
The biggest pain point for me right now is forms.  Most of them are the same, while only changing the title between templates.  These could be combined, and require passing a `title` or similar variable in the context.

**Comment by DLance96** [comment]
Now that Im finally home I should be able to work on it.

**Comment by matthewbentley** [comment]
No rush.  I'm just trying to get my thoughts down, and we still have a couple weeks.

**Comment by matthewbentley** [comment]
Another thing: automatically show the dropdown for a position when the logged in user holds that position.

**Comment by DLance96** [comment]
Im finishing up forms not sure that is the best idea if someone has 2 chair positions their top bar could get filled quickly just a thought

**Comment by matthewbentley** [comment]
Hmm, ok.  At least split the menu part into a common file that gets included on each page for that user.

**Comment by DLance96** [comment]
The way it currently is that it is section in the base.html how is that exactly different?

**Comment by matthewbentley** [comment]
Maybe I've been doing it wrong? IE for the detail manager, the detail manager dropdown shows up on the detail manager page, but not any of the detail manager sub-pages.

**Comment by DLance96** [comment]
so you should be able to make a detail manager template and have all the detail manager pages extend that it was something I had planned on doing with other positions once I realized I shouldve done it in the first place. If you have a better structure let me know.

**Comment by matthewbentley** [comment]
Yep of course I should have done that with the detail manager.  I'll fix that in a bit.
